.github/workflows/release.yml

```diff
@@ -216,7 +216,16 @@ jobs:
         if: runner.os == 'Linux' && matrix.mxe == 'none' && matrix.android == 'none' && !matrix.wasm
         run: |
           sudo apt-get update
-          sudo apt-get install libncursesw5-dev libsdl2-dev libsdl2-ttf-dev libsdl2-image-dev \
+          git clone https://github.com/libsdl-org/SDL.git
+          cd SDL
+          git checkout release-2.0.20
+          mkdir build
+          cd build
+          ../configure
```

Comment:
you can do `git clone --depth=1 https://github.com/libsdl-org/SDL.git --branch release-2.0.20` ~~turns out --branch works on tags~~
and the build instructions are simple `./configure; make; make install` https://github.com/libsdl-org/SDL/blob/release-2.0.20/INSTALL.txt#L18

Reply:
Build commands are fine

Reply:
commands work but --depth=1 and --branch will save time